website/source/guides/operations/deployment-guide.html.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: "guides"
+page_title: "Nomad Deployment Guide"
+sidebar_current: "guides-operations-deployment-guide"
+description: |-
+  This deployment guide covers the steps required to install and
+  configure a single HashiCorp Nomad cluster as defined in the
+  Nomad Reference Architecture
+product_version: 0.8
+---
+
+# Nomad Deployment Guide
+
+This deployment guide covers the steps required to install and configure a single HashiCorp Nomad cluster as defined in the [Nomad Reference Architecture](/guides/operations/reference-architecture.html).
+
+These instructions are for installing and configuring Nomad on Linux hosts running the systemd system and service manager.
+
+## Reference Material
+
+This deployment guide is designed to work in combination with the [Nomad Reference Architecture](/guides/operations/reference-architecture.html) and [Consul Deployment Guide](https://www.consul.io/docs/guides/deployment-guide.html). Although not a strict requirement to follow the Nomad Reference Architecture, please ensure you are familiar with the overall architecture design; for example installing Nomad server agents on multiple physical or virtual (with correct anti-affinity) hosts for high-availability.
```

Comment:
I would reword the last part of this paragraph to:

Although **it is** not a strict requirement to follow the Nomad Reference Architecture, please ensure you are familiar with the overall architecture design; **an example of this would be** installing Nomad server agents on multiple physical or virtual (with correct anti-affinity) hosts for high-availability.